src/common/hooks/useGenerateHash.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { sha256 } from 'js-sha256';
+
+export function useGenerate64CharHash() {
```

Comment:
Is there some specific reason this is a hook?

It doesn't access React's state nor lifecycle?

Reply:
@beganovich As we discussed on Slack, the approach has changed slightly. Instead of a hook, we simply have the function for generating the hash. Let me know your thoughts.